test/unit/MCR/teleportMCR.js

```diff
@@ -0,0 +1,99 @@
+const { expect } = require('chai');
+const { initMCR } = require('./common');
+const { ethers } = require('hardhat');
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
+
+const setup = require('./setup');
+const { setNextBlockTime } = require('../utils').evm;
+
+const { parseEther } = ethers.utils;
+
+const DEFAULT_MCR_PARAMS = {
+  mcrValue: parseEther('150000'),
+  mcrFloor: parseEther('150000'),
+  desiredMCR: parseEther('150000'),
+  mcrFloorIncrementThreshold: '13000',
+  maxMCRFloorIncrement: '100',
+  maxMCRIncrement: '500',
+  gearingFactor: '48000',
+  minUpdateTime: '3600',
+};
+
+describe('teleportMCR', function () {
+  it('teleportMCR updates values accordingly', async function () {
+    const fixture = await loadFixture(setup);
+    const { master } = fixture;
+
+    const mcr = await initMCR({ ...DEFAULT_MCR_PARAMS, master });
+
+    const { timestamp: currentTimestamp } = await ethers.provider.getBlock('latest');
+    const nextUpdateTime = currentTimestamp + 1;
```

Comment:
we need to set this deterministically to a timestamp < 1701388800 so it won't fail in the future

Reply:
will make immutable and pass a constructor argument :+1: